src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -490,32 +490,35 @@ private void clearPatronTransactionAndItemRecord(UUID itemId, InnReachTransactio
     updateItem(itemId);
   }
 
-  private Optional<InventoryItemDTO> updateItem(UUID itemId) {
-    return itemService.changeAndUpdate(itemId, item -> {
-      item.setBarcode(null);
-      return item;
-    });
-  }
+  private Optional<InventoryItemDTO> updateItem(UUID itemId){
+      return itemService.changeAndUpdate(itemId, item -> {
+        item.setBarcode(null);
+        return item;
+      });
+    }
+
+  private void verifyState(InnReachTransaction transaction, InnReachTransaction.TransactionState...states) {
+      var state = transaction.getState();
+      Assert.isTrue(ArrayUtils.contains(states, state), "Unexpected transaction state: " + state);
+    }
```

Comment:
why do you remove import for existing method InnReachTransactionUtils.verifyState and introduce a duplicate for it?